director/commands/celery.py

```diff
@@ -38,9 +38,9 @@ def worker(dev_mode, worker_args):
     """Start a Celery worker instance"""
     args = [
         "celery",
-        "worker",
         "-A",
         "director._auto:cel",
+        "worker",
```

Comment:
Same question here than the change on the "beat" command.

Reply:
see: https://github.com/ovh/celery-director/pull/137#discussion_r826368412